test/utils/image/manifest.go

```diff
@@ -40,6 +40,8 @@ type RegistryList struct {
 	GcrReleaseRegistry      string `yaml:"gcrReleaseRegistry"`
 	PrivateRegistry         string `yaml:"privateRegistry"`
 	SampleRegistry          string `yaml:"sampleRegistry"`
+	K8sCSI                  string `yaml:"k8sCSI"`
```

Comment:
Why is `K8sCSI` being introduced in this PR? Also applies to the change in the stanza below.